style.css

```diff
@@ -522,10 +514,24 @@ q:after {
 	text-rendering: auto;
 }
 
+/* Typography for Thai Font */
+
+html[lang="th"] h1, h2, h3, h4, h5, h6 {
```

Comment:
@grappler Got it. Anyway, I removed only letter-spacing property of heading, so I'm going to recheck overall. Thank you for advice :)